Lib/lib2to3/pgen2/tokenize.py

```diff
@@ -117,20 +117,20 @@ def _combinations(*l):
 )
 
 endprogs = {"'": re.compile(Single), '"': re.compile(Double),
-            "'''": single3prog, '"""': double3prog,
-            **{f"{prefix}'''": single3prog for prefix in _strprefixes},
-            **{f'{prefix}"""': double3prog for prefix in _strprefixes},
-            **{prefix: None for prefix in _strprefixes}}
+            "'''": single3prog, '"""': double3prog}
+endprogs.update({"{}'''".format(prefix): single3prog for prefix in _strprefixes})
```

Comment:
These are needed because Travis uses a old version of python for the regenerations step.